examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
OK, so in terms of this PR, anything else to add? What if I also added in a
comment in the example saying we need to notch all the channels for the
projection to work properly, which is why it's defining picks_notch.
On Dec 31, 2015 7:48 PM, "Eric Larson" <notifications@github.com> wrote:

> In examples/time_frequency/plot_compute_raw_data_spectrum.py
> <https://github.com/mne-tools/mne-python/pull/2710#discussion_r48675353>:
>
> > @@ -70,3 +71,18 @@
> >
> >  ax.set_title('Four left-temporal magnetometers')
> >  plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
> > +
> > +# Alternatively, you may also create PSDs from Raw objects with psd_XXX
> > +f, ax = plt.subplots()
> > +psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
>
> Yeah. What you have now is a bug fix and improvement for an example
> related to code you're refactoring. Let's tackle the global proj/filter
> problem more completely elsewhere, as we'll probably need a decent amount
> of discussion or changes to get it right.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2710/files#r48675353>.
>